core-services/prow/02_config/_config.yaml

```diff
@@ -2,6 +2,8 @@ branch-protection:
   allow_disabled_job_policies: true
   allow_disabled_policies: true
   orgs:
+    openshift-priv:
+      protect: false
```

Comment:
Why?

Reply:
It's not necessary (see "minimal") for the spike and I expect BP to interact with the git content sync, so I'd rather handle it separately.

Reply:
It works for `openshift`, it should work for `openshift-priv` ...